<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnalysisTool</class>
 <widget class="QWidget" name="AnalysisTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>213</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
/********************** Tab Widget ***********************/

/*
QTabWidget::pane {
	border: 2px solid rgb( 80, 80, 80 );
}
*/

/*
QTabWidget::pane{
	border-top: 2px solid #C2C7DB;
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:0.960227, stop:0 rgba(221, 221, 221, 255), stop:0.155779 rgba(228, 228, 228, 255), stop:1 rgba(202, 204, 204, 255));      
}
*/
/*
QTabBar::tab {
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:0.960227, stop:0 rgba(0, 0, 0, 0), stop:0.45 rgba(0, 0, 0, 30), stop:1 rgba(0, 0, 0, 25));
	color: white;
 }
*/

/*
QTabBar::tab:selected {
	background-color: rgb( 84, 168, 212 );
}
*/

QWidget#meanTab{
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:0.960227, stop:0 rgba(221, 221, 221, 255), stop:0.155779 rgba(228, 228, 228, 255), stop:1 rgba(202, 204, 204, 255));
}

QWidget#samplesTab{
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:0.960227, stop:0 rgba(221, 221, 221, 255), stop:0.155779 rgba(228, 228, 228, 255), stop:1 rgba(202, 204, 204, 255));      
}

QWidget#pcaTab{
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:0.960227, stop:0 rgba(221, 221, 221, 255), stop:0.155779 rgba(228, 228, 228, 255), stop:1 rgba(202, 204, 204, 255));      
}

QWidget#regressionTab{
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:0.960227, stop:0 rgba(221, 221, 221, 255), stop:0.155779 rgba(228, 228, 228, 255), stop:1 rgba(202, 204, 204, 255));      
}

/********************** Tab Widget ***********************/


/********************** Visualization Panel ***********************/

QLabel#visualizationLabel{
	color: white;
	font: bold;
}
QWidget#visualizationPanel{
	background-color: rgba( 0, 0, 0, 0 );
}

QToolButton#visualizationOpenButton{
	background-color: rgba(0, 0, 0, 0);
	border-radius: 4px;
}

QToolButton#visualizationOpenButton:hover{
	background-color: rgba( 0, 0, 0, 30 ); border-radius: 3px;
}

QWidget#visualizationHeader{
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:0.960227, stop:0 rgba(0, 0, 0, 0), stop:0.45 rgba(0, 0, 0, 30), stop:1 rgba(0, 0, 0, 25));
    border-radius: 3px;
}

QWidget#visualizationHeaderBackground{
	background-color: rgb( 84, 168, 212 );
    border-radius: 3px;
	border: 1px solid rgb(90, 90, 90);
}

QWidget#visualizationPanel{
	background-color: rgb( 90, 90, 90 );
	border-radius:4px;
	border: 1px solid rgb(60, 60, 60);
	color: rgb(90, 90, 90);
}

QWidget#visualizationContent{
	border-radius: 3px;
	border: 1px solid rgb(80, 80,80);
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:0.960227, stop:0 rgba(221, 221, 221, 255), stop:0.155779 rgba(228, 228, 228, 255), stop:1 rgba(202, 204, 204, 255));      
}
/********************** Visualization Panel ***********************/


/********************** Shape Panel ***********************/

QLabel#shapeLabel{
	color: white;
	font: bold;
}

QWidget#shapePanel{
	background-color: rgba( 0, 0, 0, 0 );
}

QToolButton#shapeOpenButton{
	background-color: rgba(0, 0, 0, 0);
	border-radius: 4px;
}

QToolButton#shapeOpenButton:hover{
	background-color: rgba( 0, 0, 0, 30 ); border-radius: 3px;
}

QWidget#layers_header_{
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:0.960227, stop:0 rgba(0, 0, 0, 0), stop:0.45 rgba(0, 0, 0, 30), stop:1 rgba(0, 0, 0, 25));
    border-radius: 3px;
}

QWidget#shapeHeaderBackground{
	background-color: rgb( 84, 168, 212 );
    border-radius: 3px;
	border: 1px solid rgb(90, 90, 90);
}

QWidget#shapePanel{
	background-color: rgb( 90, 90, 90 );
	border-radius:4px;
	border: 1px solid rgb(60, 60, 60);
	color: rgb(90, 90, 90);
}

QWidget#shapeContent{
	border-radius: 3px;
	border: 1px solid rgb(80, 80,80);
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:0.960227, stop:0 rgba(221, 221, 221, 255), stop:0.155779 rgba(228, 228, 228, 255), stop:1 rgba(202, 204, 204, 255));      
}
/********************** Shape Panel ***********************/


/********************** reconstruction Panel ***********************/

QLabel#reconstructionLabel{
	color: white;
	font: bold;
}
QWidget#reconstructionPanel{
	background-color: rgba( 0, 0, 0, 0 );
}

QToolButton#reconstructionOpenButton{
	background-color: rgba(0, 0, 0, 0);
	border-radius: 4px;
}

QToolButton#reconstructionOpenButton:hover{
	background-color: rgba( 0, 0, 0, 30 ); border-radius: 3px;
}

QWidget#reconstructionHeader{
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:0.960227, stop:0 rgba(0, 0, 0, 0), stop:0.45 rgba(0, 0, 0, 30), stop:1 rgba(0, 0, 0, 25));
    border-radius: 3px;
}

QWidget#reconstructionHeaderBackground{
	background-color: rgb( 84, 168, 212 );
    border-radius: 3px;
	border: 1px solid rgb(90, 90, 90);
}

QWidget#reconstructionPanel{
	background-color: rgb( 90, 90, 90 );
	border-radius:4px;
	border: 1px solid rgb(60, 60, 60);
	color: rgb(90, 90, 90);
}

QWidget#reconstructionContent{
	border-radius: 3px;
	border: 1px solid rgb(80, 80,80);
	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:0.960227, stop:0 rgba(221, 221, 221, 255), stop:0.155779 rgba(228, 228, 228, 255), stop:1 rgba(202, 204, 204, 255));      
}
/********************** reconstruction Panel ***********************/



/********************** Other ***********************/

QWidget#pcaWidget{
	border-radius: 3px;
	border: 2px solid rgb(80, 80,80);
}


QWidget#sampleWidget{
	border-radius: 3px;
	border: 2px solid rgb(80, 80,80);
}

QWidget#meanWidget{
	border-radius: 3px;
	border: 2px solid rgb(80, 80,80);
}


QWidget#controlsDockContents {
/*	background-color: rgb(0,0,1);
*/

/*
background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:0.960227, stop:0 rgba(1, 0, 0, 0), stop:0.45 rgba(0, 0, 1, 30), stop:1 rgba(0, 1, 0, 25));
*/

/*	background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:0.960227, stop:0 rgba(69, 168, 212, 255), stop:0.45 rgba(0, 0, 0, 30), stop:1 rgba(60, 60, 60, 255));
*/   
 border-radius: 3px;

background: rgb(90,90,90);

/*
	background: grey
*/

/*	color: red;	
*/
}



QWidget#widget {
 background-color: rgba(0,0,255,255);


/*
background-color: qlineargradient(spread:pad, x1:0.5, y1:0, x2:0.5, y2:0.960227, stop:0 rgba(1, 0, 0, 0), stop:0.45 rgba(0, 0, 1, 30), stop:1 rgba(0, 1, 0, 25));
*/

/*
	background: grey
*/

/*	color: red;	
*/
}



QToolButton{
	border: 1px solid rgba( 0, 0, 0, 0 );
	margin: 0, 0, 0, 0;
	padding: 0, 0, 0, 0;
}

QToolButton:hover{
	background-color: rgba( 140, 140, 140, 50 );
}

QToolButton:pressed{
	border: 1px solid rgba( 100, 100, 100, 120 );
	background-color: rgba( 100, 100, 100, 200 );
}




</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="allSamplesRadio">
     <property name="text">
      <string>All Samples</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="singleSamplesRadio">
     <property name="text">
      <string>Single Sample</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Sample :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sampleSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="medianButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Median</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="meanRadio">
     <property name="text">
      <string>Mean</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="pcaRadio">
     <property name="text">
      <string>PCA</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="pcaContent" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_22">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <layout class="QGridLayout" name="pcaGridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Std.Dev.</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSlider" name="pcaSlider">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>-20</number>
          </property>
          <property name="maximum">
           <number>20</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="pcaValueLabel">
          <property name="minimumSize">
           <size>
            <width>35</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QWidget" name="widget_23" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QCheckBox" name="pcaAnimateCheckBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Animate</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QWidget" name="widget_25" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="verticalSpacing">
         <number>6</number>
        </property>
        <item row="1" column="0">
         <widget class="QLabel" name="label_8">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>Eig-val.</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="pcaEigenValueLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_9">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>Lambda</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="pcaLambdaLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>24</height>
           </size>
          </property>
          <property name="text">
           <string>Mode</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="pcaModeSpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>500</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="BarGraph" name="graph_" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="regressionRadio">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Regression</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BarGraph</class>
   <extends>QWidget</extends>
   <header location="global">Analysis/bargraph.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
